nucypher/config/storages.py

```diff
@@ -296,6 +297,11 @@ class LocalFileBasedNodeStorage(NodeStorage):
     _name = 'local'
     __METADATA_FILENAME_TEMPLATE = '{}.node'
 
+    __identity_method = lambda self, x: x
```

Comment:
I'm confused by the typing here. `binascii.hexlify` which is `ENCODER` in `NodeStorage` is a 1-argument function, and `__identity_method` is a 2-argument function. How does that work?

Reply:
I initially wrote `lambda x: x`, but that raised `TypeError: <lambda>() takes 1 positional argument but 2 were given`. The problem is that we call it as `self.encoder`, which adds then additional calling argument. I'm completely open to a better design.

Reply:
I see, it seems that even lambdas in a class definition are automatically bound. Somehow I thought they're not picked up. 

I would probably go with just overloaded methods, if we have an instance variable that's also callable... Especially if we don't need to process `encoder`/`decoder` from `__init__()` (see my other comment).

Reply:
✅